docs/v1/api.md

```diff
@@ -1,63 +1,96 @@
+intent to make the service classes provided by that broker available in the
+catalog.
+
+The catalog controller has a watch on the `Broker` resource.  When the
+controller receives an `ADD` watch event for a new `Broker`, it contacts the
+broker to determine what service classes the broker offers:
+
+1. Makes a request against the given service broker's catalog endpoint
    (`GET /v2/catalog`)
-2. Translate the response to a list of `ServiceClass`es
-3. Write each translated `ServiceClass` to stable storage
+2. Translates the response to a list of `ServiceClass`
+3. Creates new `ServiceClass` instances in the API server
 
-*TODO: should we call out what happens when a `Broker` resource is deleted?*
+*TODO: What happens when a `Broker` resource is deleted?*
 
-## `ServiceClass`
+## `ServiceClass` Resource
+
+A `ServiceClass` represents an offering in the service catalog.  A
+`ServiceClass` is not usable directly; an `Instance` of a `ServiceClass` must be
+created to be consumed by an application.
 
 This resource is created by the service catalog's controller event loop after
 it has received a `Broker` resource and successfully called the backing CF
-broker's catalog endpoint. It represents a single (serviceID, planID) pair that
-a CF service broker has made available.
+broker's catalog endpoint.
+
+*TODO: Make explicit the relationship between service classes and plans*
+
+## `Instance` Resource
+
+An `Instance` represents a provisioned instance of a `ServiceClass`, and is the
+entity an application binds to.
+
+A service consumer creates an `Instance` to indicate their intent to provision
+an instance of a service class.  The `Instance` has a reference to the
+`ServiceClass` to provision an instance of.
+
+The service catalog controller has a watch on the `Instance` resource and
+receives an `ADD` watch event. The controller then attempts to provision a new
+instance:
 
-*TODO: what happens in the below cases?*
+1.  The controller calls the provision endpoint on the broker 
+2.  The broker returns a response indicating that the broker provisioned the new
+    instance, the instance was already provisioned, or the provisioning
+    operation is in progress
+3.  The controller updates the status of the `Instance` to indicate when the
+    `Instance` is in a provisioning or provisioned condition
 
-1. *The `Broker` that caused the `ServiceClass` was deleted*
-1. *The `ServiceClass` itself was deleted*
+*TODO: codify how asynchronous responses are handled in the controller*
 
-## `Instance`
+## `Binding` Resource
 
-This resource is created by a service consumer to indicate their desire to
-provision a service. When the service catalog's controller event loop sees an
-`Instance` created, it calls the provision endpoint on the backing CF service
-broker and writes `provisioned` into a `status` field of the
-`Instance`.
+A `Binding` represents the binding between an application and an `Instance` of a
```

Comment:
perhaps "... represents a "used by" relationship between... "
or "... represents a link between ..."
just trying to avoid using the word "bind" again.